### PR TITLE
some fixes for #277

### PR DESCRIPTION
**Pull request by reidrankin**
This is a rollup of the suggestions in my review on #277; I'll copy the comments over as appropriate.

### CODE REVIEW

File: lib/firmware/fsm_msg_tendermint.h [hunk 1]
```diff
@@ -154,7 +154,9 @@ void fsm_msgTendermintMsgAck(const TendermintMsgAck *msg) {
     case OutputAddressType_TRANSFER:
     default: {
       char amount_str[32];
-      bn_format_uint64(msg->send.amount, NULL, msg->denom, 6, 0, false, amount_str,
+      char suffix[sizeof(coin->coin_shortcut) + 1];
+      snprintf(suffix, sizeof(suffix), " %s", coin->coin_shortcut);
+      bn_format_uint64(msg->send.amount, NULL, suffix, coin->decimals, 0, false, amount_str,
```
Comment (reidrankin) [inline lib/firmware/fsm_msg_tendermint.h:159]: on <=7.1.4 you'd get a confirm message like `0.01 ATOM`, but after #277 you'd get `0.01uatom`.

File: include/keepkey/transport/messages-tendermint.options [hunk 2]
```diff
@@ -1,22 +1,22 @@
 TendermintGetAddress.address_n                             max_count:10
-TendermintGetAddress.address_prefix                        max_size:10
-TendermintGetAddress.chain_name                            max_size:15
+TendermintGetAddress.address_prefix                        max_length:9
+TendermintGetAddress.chain_name                            max_length:14
 
-TendermintAddress.address                                  max_size:46
+TendermintAddress.address                                  max_length:53
```
[the remark this answers is not in the document]
Comment (markrypto) [reply include/keepkey/transport/messages-tendermint.options:5]: cool

File: lib/firmware/fsm_msg_tendermint.h (hunk 1 above)
Comment (markrypto) [reply lib/firmware/fsm_msg_tendermint.h:159]: Isn't uatom correct in this case?

Comment (reidrankin) [reply lib/firmware/fsm_msg_tendermint.h:159]: not if you divide by 10^6 first!

Comment (markrypto) [reply lib/firmware/fsm_msg_tendermint.h:159]: cool

**Review by markrypto — APPROVED** [review]

**Comment by reidrankin** [comment]
closing in favor of #288

**Pull request closed without merging**